### PR TITLE
Move functionality from AClassWrapper to AClass

### PR DESCRIPTION
**Pull request by mernst**

### CODE REVIEW

**Comment by mernst** [comment]
@kelloggm Please review.  Thanks!

File: scene-lib/src/scenelib/annotations/el/AClass.java [hunk 1]
```diff
@@ -224,4 +251,143 @@ public boolean isEmptyValue(AExpression v) {
...
+        return tlAnnotationsHere;
+    }
+
+    /**
+     * Update the method's fields, using information from the given ExecutableElement.
+     *
+     * @param methodElt the method
+     * @return an interned AMethod representing the method
+     */
+    public AMethod setFieldsFromMethodElement(ExecutableElement methodElt) {
```
Comment (kelloggm) [inline scene-lib/src/scenelib/annotations/el/AClass.java]: The name and/or Javadoc of this method should indicate that the method associated with the passed `MethodElement` is vivified by calling this method.

**Review by kelloggm — APPROVED** [review]
Two minor comments on the docs, otherwise looks good.

File: scene-lib/src/scenelib/annotations/el/AClass.java [hunk 2]
```diff
@@ -224,4 +251,143 @@ public boolean isEmptyValue(AExpression v) {
...
+        String methodSignature = JVMNames.getJVMMethodSignature(methodElt);
+        AMethod method = methods.getVivify(methodSignature);
+        method.setFieldsFromMethodElement(methodElt);
+        return method;
+    }
+
+    /**
+     * Set the type mirror of the given field.
```
Comment (kelloggm) [inline scene-lib/src/scenelib/annotations/el/AClass.java]: Likewise, this method should indicate that the field is vivified.

**Comment by mernst** [comment]
@kelloggm I don't have permission to your repository, so I cannot merge this pull request.

**Pull request merged**